airbyte-ci/connectors/metadata_service/lib/metadata_service/models/src/Rollout.yaml

```diff
@@ -0,0 +1,25 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
+"$id": https://github.com/airbytehq/airbyte/airbyte-ci/connectors/metadata_service/lib/metadata_service/models/src/Rollout.yaml
+title: Rollout
+description: configuration for the rollout of a connector
+type: object
+additionalProperties: false
+properties:
+  initialPercentage:
+    type: integer
+    minimum: 0
+    maximum: 100
+    default: 0
+    description: The percentage of users that should receive the new version initially.
+  maxPercentage:
+    type: integer
+    minimum: 0
+    maximum: 100
+    default: 100
```

Comment:
WDYT about defaulting this to 50 instead? That would mean that up to half of the connectors will get the RC before it gets fully rolled out.